src/equations/compressible_euler_2d.jl

```diff
@@ -1907,11 +1907,11 @@ end
     v1 = rho_v1 / rho
     v2 = rho_v2 / rho
     v_square = v1^2 + v2^2
-    inv_rho_gammap1 = (1 / rho)^(equations.gamma + 1.0)
+    inv_rho_gammap1 = (1 / rho)^(equations.gamma + 1)
 
     # The derivative vector for the modified specific entropy of Guermond et al.
     w1 = inv_rho_gammap1 *
-         (0.5 * rho * (equations.gamma + 1.0) * v_square - equations.gamma * rho_e)
+         (0.5f0 * rho * (equations.gamma + 1) * v_square - equations.gamma * rho_e)
```

Comment:
Could you please let others follow the docs when making PRs? I cannot always check the incoming functions as I am moving forward with my work.